geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/CreateIndexCommand.java

```diff
@@ -78,15 +79,15 @@ public Result createIndex(@CliOption(key = CliStrings.CREATE_INDEX__NAME, mandat
     AtomicReference<XmlEntity> xmlEntity = new AtomicReference<>();
 
     try {
-      int idxType;
+      IndexType idxType;
 
       // Index type check
       if ("range".equalsIgnoreCase(indexType)) {
-        idxType = IndexInfo.RANGE_INDEX;
+        idxType = IndexType.FUNCTIONAL;
```

Comment:
This might read a little nicer with a Stream: 

```
   IndexType idxType = Arrays.stream(IndexType.values())
            .filter(type -> type.getDescription().equalsIgnoreCase(indexType))
            .findFirst()
            .orElse(null);
      
     if (idxType == null) {
       return ResultBuilder
           .createUserErrorResult(CliStrings.CREATE_INDEX__INVALID__INDEX__TYPE__MESSAGE);
     }
```

Reply:
this will get deleted with the next PR by changing the parameter type from String to this enum IndexType for better client side validation.